run.py

```diff
@@ -31,7 +34,8 @@
 parser.add_argument('-t', '--target', help='replace this face', dest='target_path')
 parser.add_argument('-o', '--output', help='save output to this file', dest='output_file')
 parser.add_argument('--keep-fps', help='maintain original fps', dest='keep_fps', action='store_true', default=False)
-parser.add_argument('--gpu', help='use gpu', dest='gpu', action='store_true', default=False)
+if torch.cuda.is_available():
```

Comment:
This isn't a good idea. I have an AMD GPU, I don't have CUDA - I have ROCM.

Reply:
From what I found in the internet this should still work for Radeon GPUs, can you give it a try?